integration_tests/e2e/find.cy.ts

```diff
@@ -23,7 +23,10 @@ context('Find', () => {
     cy.visit(findPaths.courses.index({}))
 
     const coursesPage = Page.verifyOnPage(CoursesPage)
-    coursesPage.shouldHaveCourses(courses)
+
+    const sortedCourses = [...courses].sort((courseA, courseB) => courseA.name.localeCompare(courseB.name))
+
+    coursesPage.shouldHaveCourses(sortedCourses)
```

Comment:
I wonder if `shouldHaveCourses` should be doing the sorting, given the method is specific to this page and we're making a default assumption about this page?